content/en/community/hackathons/sessions/baby-steps/content/Demos_02.-Building-and-Running-the-httpreply-Application.md

```diff
@@ -99,6 +99,14 @@ oOo oOO| | | | |   (| | | (_) |  _) :_
 Listening on port 8123...
 ```
 
+In this step you may face the following error:
+```text
+access denied by acl file
+qemu-system-ppc64: -netdev bridge,br=virbr0,id=net0: bridge helper failed
+```
+
+To fix this, check if the `/etc/qemu/bridge.conf` file exists. If it doesn't, then create it first.and then add the line `allow virbr0` to it to allow your bridge to run.
```

Comment:
```suggestion
To fix this, check if the `/etc/qemu/bridge.conf` file exists.
If it doesn't, then create it first and then add the line `allow virbr0` to it to allow your bridge to run.
```

Reply:
@levihackerman-102, if you add `allow all`, would that work?

Reply:
Yes @razvand, it works.